docs/README.md

```diff
@@ -1,49 +1,52 @@
-# Docs
+# The OPA Website and Documentation
 
-This directory contains all of the Markdown, HTML, CSS, and other assets needed
-to build the [openpolicyagent.org](http://openpolicyagent.org) site. See the
+This directory contains all of the Markdown, HTML, Sass/CSS, and other assets needed
+to build the [openpolicyagent.org](https://openpolicyagent.org) website. See the
 section below for steps to build the site and test documentation changes
 locally.
 
-The raw OPA documentation can be found under the [book](./book) directory.
+The raw OPA documentation can be found under the [content/docs](./content/docs)
+directory.
 
-For development documentation see the [devel](./devel) directory.
+For documentation on developing OPA, see the [devel](./devel) directory.
 
-## Site Updates
-
-We use GitHub pages to host the website that includes all of the OPA
-documentation. In order to update the website, you need to have write permission
-on the open-policy-agent/opa repository.
-
-### Prerequisites
+## Prerequisites
 
 If you want to build and serve the site locally, you need the following packages
 installed on your system:
 
-- npm
-- [gitbook](https://github.com/GitbookIO/gitbook)
+- [npm](https://npmjs.org)
+- The [Hugo](https://gohugo.io/getting-started/installing/) static site generator
```

Comment:
I installed Hugo v0.54.0 but I'm getting this error:

```
$ make serve
hugo server \
	--buildDrafts \
	--buildFuture \
	--disableFastRender
Building sites … ERROR 2019/03/21 08:57:41 error: failed to transform resource: TOCSS: failed to transform "sass/style.sass" (text/x-sass): this feature is not available in your current Hugo version
Total in 330 ms
Error: Error building site: logged 1 error(s)
Makefile:5: recipe for target 'serve' failed
make: *** [serve] Error 255
```

Reply:
I was able to resolve this by cloning the Hugo repo and building from source with `--tags extended`:

```
go build --tags extended
```

Note, I had to upgrade the Go version on my development machine to 1.12.

@lucperkins could you figure out how to make this step a bit more reproducible?

Reply:
@tsandall Yeah, I can add OS-specific docs around that.